src/features/transactions/components/positions.tsx

```diff
@@ -43,7 +43,11 @@ export function Positions({ type }: PositionsTableProps) {
   if (data.length === 0) {
     return (
       <EmptyMessage
-        message={t('transactions.no_open_positions')}
+        message={
+          type === 'open'
+            ? t('transactions.no_open_positions')
+            : t('transactions.no_closed_positions')
+        }
```

Comment:
I already did this in some PR (I didn't wait for your approval, sorry, but it was so minor I didn't want to bother). Update your branch and leave only the truncation part of the PR